yac.c

```diff
@@ -113,6 +113,12 @@ PHP_INI_BEGIN()
     STD_PHP_INI_ENTRY("yac.values_memory_size", "64M", PHP_INI_SYSTEM, OnChangeValsMemoryLimit, v_msize, zend_yac_globals, yac_globals)
     STD_PHP_INI_ENTRY("yac.compress_threshold", "-1", PHP_INI_SYSTEM, OnChangeCompressThreshold, compress_threshold, zend_yac_globals, yac_globals)
     STD_PHP_INI_ENTRY("yac.enable_cli", "0", PHP_INI_SYSTEM, OnUpdateBool, enable_cli, zend_yac_globals, yac_globals)
+#if ENABLE_MSGPACK
+	/* use msgpack when available to keep previous behavior */
+    STD_PHP_INI_ENTRY("yac.serializer", "1", PHP_INI_SYSTEM, OnUpdateLong, serializer, zend_yac_globals, yac_globals)
+#else
+    STD_PHP_INI_ENTRY("yac.serializer", "0", PHP_INI_SYSTEM, OnUpdateLong, serializer, zend_yac_globals, yac_globals)
+#endif
```

Comment:
INI_SYSTEM as probably dangerous to change it in a script, which may result in corrupted memory (half being serialized with php, half with msgpack).